utils/common/src/main/kotlin/DirectoryStash.kt

```diff
@@ -32,24 +32,27 @@ import kotlin.io.path.moveTo
 import org.apache.logging.log4j.kotlin.Logging
 
 /**
- * A convenience function that stashes directories using a [DirectoryStash] instance.
+ * A convenience function that stashes [directories] to the [stashParentDir] using a [DirectoryStash] instance.
```

Comment:
commit-msg: Please explain shortly the problem this solves and why it solved it.

(Would it make sense to split-up this commit into two?)

Reply:
I'm now using a completely different approach, and the last commit should explain the details better.